util/gzip_compression.go

```diff
@@ -0,0 +1,34 @@
+package util
```

Comment:
Put this in its own sub-package of util, so that it can be included by itself instead of picking up other stuff in the same package.

Reply:
good call, fixed it